ui/docker-compose.yaml

```diff
@@ -3,6 +3,9 @@ services:
   ui:
     build:
       context: ./src/dev/server
+      args:
+        - UID=${UID}
+        - GID=${GID}
```

Comment:
This seems to make this image usable only on the machine on which it was built. I'm not judging, just checking if that is true.

Reply:
That assessment is correct; it'd be nice to avoid this side effect, though we're not currently planning to publish or reuse these images (i.e. `docker-compose build` is an essential step).